### PR TITLE
feat: Add react_Image widget

### PR DESCRIPTION
**Pull request by jbrundage**
Signed-off-by: Jaman Brundage <jbrundage372@gmail.com>

## Checklist:
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [x] The commit message includes a "fixes" reference if appropriate.
  - [x] The commit is signed.
- [ ] The change has been fully tested:
  - [ ] I have viewed all related gallery items
  - [ ] I have viewed all related dermatology items
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised new issues to address them separately

### CODE REVIEW

File: packages/react/src/Image.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    y?: number;
+    height?: number;
+    width?: number;
+    href?: string;
+    text?: string;
+    yOffset?: number;
+}
+
+export const Image: React.FunctionComponent<IImage> = ({
+    x,
```
Comment (GordonSmith) [inline packages/react/src/Image.tsx:14]: Should x have a default value (should it be optional?)

Comment (jbrundage) [reply packages/react/src/Image.tsx:14]: Yes I think so, that prevents ever seeing the svg "x is NaN" error.

Comment (GordonSmith) [reply packages/react/src/Image.tsx:14]: FYI I "think" React will spot an undefined and omit the entire assignment - so that wouldn't be the case.

**Comment by jbrundage** [comment]
@GordonSmith this has been refactored in the way that we discussed

File: demos/gallery/samples/graph2/TIN.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+function getDemoImageURL(w, h, color) {
+    const canvas = document.createElement('canvas');
+    canvas.width = w;
+    canvas.height = h;
+    const ctx = canvas.getContext('2d');
+    ctx.fillStyle = color;
+    ctx.fillRect(0,0,w,h);
+    return canvas.toDataURL();
+}
```
Comment (GordonSmith) [inline demos/gallery/samples/graph2/TIN.js:37]: Rename file to human readable name (the filename ends up in the UI)

File: packages/react/src/Image.tsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    text?: string;
+}
+
+export const Image: React.FunctionComponent<IImage> = function ({
+    href,
+    text,
+    width,
+    height
+}) {
+    const title = typeof text !== "undefined" ? <title>{text}</title> : undefined;
```
Comment (GordonSmith) [inline packages/react/src/Image.tsx:16]: Shorten to:  `text !== undefined`

**Comment by jbrundage** [comment]
@GordonSmith these changes have been made

**Pull request closed without merging**